packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 12.0.0
+
+* Adds error handling on Flutter Api methods.
+* **Breaking Change** Removes `Reply` class from all Java method returns.
```

Comment:
Did we stop using the `[language]` tags on release notes on purpose? I only just noticed we haven't been doing it recently; it seems convenient for people skimming for things that do or don't apply to their use cases.